src/assets/scss/blog.scss

```diff
@@ -153,3 +153,63 @@ ul.tag-list {
 .from-the-blog .blog-posts {
 	grid-column: 1 / -1;
 }
+
+.code-wrapper {
+	display: flex;
+	align-items: start;
+	justify-content: space-between;
+	margin: 1.5rem 0;
+	border-radius: var(--border-radius);
+	background-color: var(--lightest-background-color);
+
+	> pre {
+		margin: 0;
+	}
+}
```

Comment:
![Screenshot 2025-03-11 180544](https://github.com/user-attachments/assets/9d1a5d0e-0353-4fa5-b874-00041d626c78)

I not sure if this is the right approach, we can just add some padding (`padding-right: 3.5rem`) to `pre` tag
https://github.com/eslint/eslint.org/blob/1b26995f615351e3f969c76b5bbc2486a6f56ab5/src/assets/scss/syntax-highlighter.scss#L30-L37

so can you please revert this change.

and also i think we can keep the background always not just on hover to make it more visible. 
![Screenshot 2025-02-28 185434](https://github.com/user-attachments/assets/ff025b18-12f9-43de-a4cb-6fab7bc33d65)

Reply:
Agree with @Tanujkanti4441. The background color makes it a lot more visible. :+1: